API/DTOs/ChapterDto.cs

```diff
@@ -43,6 +43,10 @@ public class ChapterDto : IHasReadTimeEstimate
     /// </summary>
     public int PagesRead { get; set; }
     /// <summary>
+    /// Last time the progress was synced from UI or external app
```

Comment:
This comment doesn't make sense for within Kavita. It would be more correct to say, Last time Chapter was read in Utc.

Reply:
ok